functions/src/modules/pm/categories/prizes/index.ts

```diff
@@ -2,7 +2,7 @@ import type { Express } from "express";
 import createPrize from "./createPrize";
 import deletePrize from "./deletePrize";
 import getPrize from "./getPrize";
-import listPrizes from "./listPrizes";
+import listPrizes from "./listPrize";
```

Comment:
This endpoint should be `listPrizes`...
```suggestion
import listPrizes from "./listPrizes";
```